google/ggrc-core
# GGRC-5880/GGRC-7116/Latency while downloading Assessment csv import template

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] GGRC-7116
- [x] #9651 

# Issue description

Downloading Assessment template csv. file took more then 30 sec.

# Steps to test the changes

- Open Import page
- Click on 'Download Import Template' > Select 'Assessment' object type from the list
- Click on 'Download CSV'

Actual behavior: Import Assessment csv template downloaded during ~ 27 sec
Expected behavior: Improve performance for downloading csv import template with choosing at Import page (step 2) an import template with only LCA that refer to a concrete Assessment template.

# Solution description

Add additional input field with autocomplete to select specific assessment template. Send list of required templates to the server. 

# Sanity checklist




# PR Review checklist
